pkgs/development/python-modules/packaging/default.nix

```
@@ -10,12 +10,12 @@
 
 buildPythonPackage rec {
   pname = "packaging";
-  version = "20.9";
+  version = "21.2";
   format = "pyproject";
```

```suggestion
  version = "21.2";
  format = "pyproject";
  
  disabled = pythonOlder "3.6";
```

see https://github.com/pypa/packaging/blob/main/setup.py#L50